manim/utils/testing/_frames_testers.py

```diff
@@ -11,6 +11,10 @@
 from ._show_diff import show_diff_helper
 
 
+FRAME_ABSOLUTE_TOLERANCE = 1.01
+FRAME_NUMBER_MISMATCH_TOLERANCE = 1e-5
```

Comment:
```suggestion
FRAME_MISMATCH_RATIO_TOLERANCE = 1e-5
```

Ok, I'm **_really_** nitpicking, it should be

Reply:
True! d05382db